model/src/main/scala/za/co/absa/atum/model/dto/AdditionalDataSubmitDTO.scala

```diff
@@ -0,0 +1,6 @@
+package za.co.absa.atum.model.dto
+
+case class AdditionalDataSubmitDTO (
```

Comment:
Perhaps the AdditionalDataDTO that I created in the past can be removed even

Reply:
don't bother, I change it here: https://github.com/AbsaOSS/atum-service/pull/143 with all the tests etc, so please don't waste your time to do it also